spec/cases_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'spec_helper'
 require 'json'
 
-describe MessagePack do
+RSpec.describe MessagePack do
```

Comment:
Is `Rspec.describe` recommended? I can't find about this syntax at http://betterspecs.org/#describe